scripts/deploy_assisted_service.sh

```diff
@@ -15,6 +15,19 @@ export OCP_SERVICE_PORT=$(( 7000 + $NAMESPACE_INDEX ))
 export OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE:-}
 export ENABLE_KUBE_API=${ENABLE_KUBE_API:-false}
 export ENABLE_KUBE_API_CMD="ENABLE_KUBE_API=${ENABLE_KUBE_API}"
+export OPENSHIFT_VERSIONS=${OPENSHIFT_VERSIONS:-}
+export OPENSHIFT_VERSIONS_CMD=""
+
+if [[ "${ENABLE_KUBE_API}" == "true" && -z "${OPENSHIFT_VERSIONS}" ]]; then
```

Comment:
Maybe we should put it inside https://github.com/openshift/assisted-test-infra/blob/f47f6d9e3706cb786ce171a9300c6e39d231ca85/Makefile#L111
That way all things related to KUBE_API are in one place

Reply:
You mean moving the json parse logic to the Makefile? Not sure it'll be cleaner as it would clutter the Makefile:/
Also, this logic is relevant only for the deploy service flow, so I guess it's a proper place for having context.